### ISSUE TITLE
Better support for non-English books 

### ISSUE THREAD

**Issue by HoratioVex** [btpf/Alexandria#42]
I suggest a few additions to aid with books in other languages:

* A UI option to override the "lang" attribute for the current book.
Many books don't set their language correctly, preventing the other features from working.
* A UI option to override the "hyphens" CSS property for the current book to toggle automatic hyphenation.
Languages with longer words make more aggressive hyphenation desirable, but ultimately it's a matter of taste that should be left to the user.
* Use the correct dictionary for the set language, if available.

**Comment by Korb** [comment]
> A UI option to override the "hyphens" CSS property for the current book to toggle automatic hyphenation. Languages with longer words make more aggressive hyphenation desirable, but ultimately it's a matter of taste that should be left to the user.

I agree that non-English texts, especially when justified columns are enabled, without word hyphenations often look sloppy and take up more space than they otherwise would. Forcing you to scroll more often. Especially when the text is displayed in columns.
